### ISSUE TITLE
[Task] investigate error on production with v2.3.0

### ISSUE THREAD

**Issue by OriHoch** [Midburn/spark#339]
after failed deployment of v2.3.0 I see this error in logs of one of the failed instances

no sure if it's the cause of the problem, but worth to investigate
```
 #011{"isFulfilled":false,"isRejected":true,"rejectionReason":{"message":"Cannot read property 'email' of undefined","stack":"TypeError: Cannot read property 'email' of undefined\n    at _after_update (/opt/spark/latest/routes/api_camps_routes.js:129:100)\n    at tryCatcher (/opt/spark/latest/node_modules/bluebird/js/release/util.js:16:23)\n    at Promise._settlePromiseFromHandler (/opt/spark/latest/node_modules/bluebird/js/release/promise.js:512:31)\n    at Promise._settlePromise (/opt/spark/latest/node_modules/bluebird/js/release/promise.js:569:18)\n    at Promise._settlePromise0 (/opt/spark/latest/node_modules/bluebird/js/release/promise.js:614:10)\n    at Promise._settlePromises (/opt/spark/latest/node_modules/bluebird/js/release/promise.js:693:18)\n    at Async._drainQueue (/opt/spark/latest/node_modules/bluebird/js/release/async.js:133:16)\n    at Async._drainQueues (/opt/spark/latest/node_modules/bluebird/js/release/async.js:143:10)\n    at Immediate.Async.drainQueues (/opt/spark/latest/node_modules/bluebird/js/release/async.js:17:14)\n    at runCallback (timers.js:651:20)\n    at tryOnImmediate (timers.js:624:5)\n    at processImmediate [as _immediateCallback] (timers.js:596:5)"}}
```

**Comment by asihud** [comment]
this might get the instance to crush, ill check that.
can i have access to the all log?
further more, is there any way to do upload releases easily? every crash like that , that require to upload a version quickly, makes it hard to manage.

**Comment by asihud** [comment]
this issue was solved at v2.3.1.

**Comment by OriHoch** [comment]
thanks, opened follow-up issue: #345
